core/lib/src/main/java/org/opennms/core/utils/PropertiesCache.java

```diff
@@ -55,17 +60,19 @@
 	private static final Logger LOG = LoggerFactory.getLogger(PropertiesCache.class);
 
     public static final String CHECK_LAST_MODIFY_STRING = "org.opennms.utils.propertiesCache.enableCheckFileModified";
+    public static final String CACHE_TIMEOUT = "org.opennms.utils.propertiesCache.cacheTimeout";
+    public static final int DEFAULT_CACHE_TIMEOUT = 600;
```

Comment:
Can we increase the default timeout to one hour instead of 10 minutes? I don't think we need to be too aggressive here given that we never used to evict these.

Reply:
done!